.github/workflows/dockerhub.yml

```diff
@@ -31,10 +31,10 @@ jobs:
     - name: Push docker images with version tag to dockerhub
       if: startsWith(github.ref, 'refs/tags/')
       run: |
-        echo "DOCKER_IMAGE_VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
-
+        export DOCKER_IMAGE_VERSION="${{ github.ref_name }}"
         make dockerpush-only
         make dockerpush-bs-only
```

Comment:
why do this twice?

Reply:
Once it pushes to `go-spacemesh` and once to `go-spacemesh-dev`.

The idea is that `go-spacemesh-dev` contains every build (also the ones used for `bors try`, `bors merge`, etc.) while `go-spacemesh` only contains released versions of the node.